modules/troubleshooting-disabling-autoreboot-mco-console.adoc

```diff
@@ -0,0 +1,96 @@
+// Module included in the following assemblies:
+//
+// * support/troubleshooting/troubleshooting-operator-issues.adoc
+
+[id="troubleshooting-disabling-autoreboot-mco-console_{context}"]
+= Disabling Machine Config Operator from automatically rebooting by using the console
+
+To avoid unwanted disruptions from changes made by the Machine Config Operator (MCO), you can use the {product-title} web console to modify the machine config pool (MCP) to prevent the MCO from making any changes to nodes in that pool. This prevents any reboots that would normally be part of the MCO update process. 
+
+[NOTE]
+====
+Pausing a machine config pool stops all updates to your {op-system} nodes, including updates to the operating system, security, certificate, and any other updates related to the machine config. Pausing should be done for short periods of time only.
+====
+
+.Prerequisites
+
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+To pause or unpause automatic MCO update rebooting: 
+
+* Pause the autoreboot process:
+
+. Log into the {product-title} web console as a user with the `cluster-admin` role.
+
+. Click *Compute* -> *Machine Config Pools*.
+
+. On the *Machine Config Pools* page, click either *master* or *worker* depending upon which nodes you want to pause rebooting.
+
+. On the *master* or *worker* page, click *YAML*.
+
+. In the YAML field, update the `spec.paused` field to `true`.
++
+.Sample `MachineConfigPool` object 
+[source,yaml]
+----
+apiVersion: machineconfiguration.openshift.io/v1
+kind: MachineConfigPool
+...
+
+spec:
+
+...
+  paused: true <1>
+----
+<1> Update the `spec.paused` field to `true` to pause rebooting.
+
+. To verify that the machine config pool is paused, return to the *Machine Config Pools* page.
++
+On the *Machine Config Pools* page, the *Paused* column reports *True* for the MCP you modified.
++
+If the machine config pool has pending changes while paused, the *Updated* column is *False* and *Updating* is *False*. When *UPDATED* is *True* and *UPDATING* is *False*, there are no pending changes.
++
+[IMPORTANT]
+====
+If there are pending changes (*UPDATED=False* and *UPDATING=False*), it is recommended to schedule a maintenance window for a reboot as early as possible, using the following steps, so that the queued changes since last reboot can take effect.
```

Comment:
Does the parenthetical mean both are false specifically, or is that but one example? I could read it two ways:

- If there are pending changes, such as when both **UPDATED** and **UPDATING** are **False**, 
- If there are pending changes, where both **UPDATED** and **UPDATING** are **False**,

Reply:
This sentence is pretty long, so I'd suggest breaking it after "as possible".
And for "using the following steps" - can you be more specific here? Maybe something like, "Use the following steps for unpausing autoreboot to apply the changes that were queued since the last reboot."